assets/scss/main.scss

```diff
@@ -3,3 +3,9 @@
 @import "./vendor";
 @import "./layout";
 @import "./components";
+
+
+:root {
+  --cds-focus: #{$purple-70};
+  --cds-interactive-01: #{$purple-70};
+}
```

Comment:
I believe that we should preferably leave only one empty line between the blocks and add the EOF new line.

Reply:
fixed with https://github.com/Qiskit/qiskit.org/pull/3020/commits/5238a9b32d9395094f7bccc4eb9da296db4a842c